spec/models/manageiq/providers/redhat/infra_manager/refresh/refresher_4_async_graph_spec.rb

```diff
@@ -36,7 +36,7 @@ def load_response_mock_for(filename)
 
   it "will perform a full refresh on v4.1" do
     EmsRefresh.refresh(@ems)
-    VCR.use_cassette("#{described_class.name.underscore}_ovn_provider") do
+    VCR.use_cassette("#{described_class.name.underscore}_ovn_provider", :allow_unused_http_interactions => true) do
```

Comment:
@AlonaKaplan for some reason it fails without the :allow_unused_http_interactions, any idea why that could be?

Reply:
specifically on:
       There are unused HTTP interactions left in the cassette:
         - [get http://alkaplan1.usersys.redhat.com:9696/] => [200 "{\"versions\": [{\"status\": \"CURRENT\", \"id\": \"v2.0\", \"links\": [{\"href\": \"http://alk"]

Reply:
so make sure that vcr tape contains those calls. If we add it it is almost like having no test.

Reply:
The vcr tape contains them, the problem is it contains also other calls we do not need, but this cassette is shared with other tests and I do not know why the ovn would do less calls here.
Anyway I do not think this is like not having a test, this spec actually makes sure everything works.

Reply:
In this case seems like sharing is not the best idea. I am thinking here about future changes and how we are going to tack whether there are some unneeded calls that should not be there.

Reply:
Hi Boris, there was a bug causing the ovn test to fail (ManageIQ/manageiq-providers-ovirt/pull/231). Are you sure that now the fix is merged ':allow_unused_http_interactions' is still needed here?

I don't like the approach of adding this flag. If actually there are cases where less interactions/calls are needed (which I don't think is the case here), a new cassette with the relevant interactions should be recorded.